ScreenCapture/ScreenCapture.cpp

```diff
@@ -334,6 +345,117 @@ namespace WPEFramework
         }
 #endif
 
+#ifdef HAS_FRAMEBUFFER_API_HEADER
+        static vnc_bool_t FakeVNCServerFramebufferUpdateReady(void* ctx) {
+            LOGWARN("FakeVNCServerFramebufferUpdateReady called");
+            return vnc_true;
+        }
+
+        static void FakeVNCServerFramebufferDetailsChanged(void* ctx, vnc_uint8_t* fb, vnc_uint16_t width, vnc_uint16_t height, vnc_uint16_t stride, PixelFormat* pf) {
+            LOGWARN("FakeVNCServerFramebufferDetailsChanged called");
+        }
+
+        static void FakeVNCServerPaletteChanged(void* ctx, Palette* palette) {
+            LOGWARN("FakeVNCServerPaletteChanged called");
+        }
+
+        static void FakeVNCServerLogMessage(void* ctx_, const char* fmt, ...) {
+            LOGWARN("VNCServerLogMessage called");
+        }
+
+        bool ScreenCapture::getScreenshotRealtek(std::vector<unsigned char> &png_out_data)
+        {
+            ErrCode err;
+            vnc_bool_t result;
+            vnc_uint8_t* buffer; 
+            VncServerFramebufferAPI api;
+
+            api.framebufferUpdateReady = FakeVNCServerFramebufferUpdateReady;
+            api.framebufferDetailsChanged = FakeVNCServerFramebufferDetailsChanged;
+            api.paletteChanged = FakeVNCServerPaletteChanged;
+            api.logMsg = FakeVNCServerLogMessage;
+
+            FBContext *context = NULL;
+            err = fbCreate(&context);
+            if (err != ErrNone) {
+                LOGERR("fbCreate fail");
+                return false;
+            }
+
+            err = fbInit(context, &api, NULL);
+            if (err != ErrNone) {
+                LOGERR("fbInit fail");
```

Comment:
One more possible memory leak left here, in case fbInit() fails. It probably will be super rare, but still.

Reply:
Fixed this one as well.